open-backport-pull-request/src/run.ts

```diff
@@ -22,37 +20,30 @@ type Outputs = {
 }
 
 export const run = async (inputs: Inputs): Promise<Outputs | undefined> => {
-  const octokit = getOctokit(inputs.githubToken, octokitPluginRetry.retry)
-  const { baseBranch, headBranch } = inputs
-
-  if (await hasDiff({ headBranch, baseBranch, octokit, context })) {
-    return await openPullRequest(
-      {
-        headBranch,
-        baseBranch,
-        skipCI: inputs.skipCI,
-        mergePullRequest: inputs.mergePullRequest,
-        context,
-      },
-      octokit,
-    )
-  }
-}
+  const octokit = github.getOctokit(inputs.githubToken, {}, octokitPluginRetry.retry)
```

Comment:
It needs to pass a plugin to third argument.

```ts
export function getOctokit(
  token: string,
  options?: OctokitOptions,
  ...additionalPlugins: OctokitPlugin[]
): InstanceType<typeof GitHub> {
```

https://github.com/actions/toolkit/blob/ae38557bb0dba824cdda26ce787bd6b66cf07a83/packages/github/src/github.ts#L15-L19